### PR TITLE
Add devices-overview page app

### PR DESCRIPTION
**Pull request by psi-4ward**

### CODE REVIEW

**Comment by psi-4ward** [comment]
I'll remove the `docs/CNAME` so github will build and deploy my fork. Can you remember to add it again after merge?

**Comment by Koenkk** [comment]
I'm wondering if the flow can be streamlined a bit (reduce amount of files and amount of locations of the supported devices object). Would the following be possible:
- `devices-overview/` contains the app, upon a `npm run build` 1 HTML file is created: `dist/device-overview.html`, this file has the js, css code and fonts inlined, so it is completely standalone. This will be the only generated file that will be checked-in.
- `dist/device-overview.html` has a placeholder where the device object should be inserted, e.g. `window.ZIGBEE2MQTT_SUPPORTED_DEVICES = %ZIGBEE2MQTT_SUPPORTED_DEVICES%`
- Upon running `npm run docgen`, a copy is made of the `dist/device-overview.html` and the `%ZIGBEE2MQTT_SUPPORTED_DEVICES%` placeholder is replaced with the actual devices object.

File: test/docgen.test.js [hunk 1]
```diff
@@ -4,15 +4,18 @@ const path = require('path');
 
 const base = path.join(__dirname, '..', 'docs');
 
-const supportDevices = require('../docgen/supported-devices');
+const supportedDevices = require('../docgen/supported-devices');
 
 describe('Docgen', () => {
-    it('supported-devices.md should be up-to-date.', () => {
+    it('supported-devices.md should be up-to-date.', (done) => {
```
[the remark this answers is not in the document]
Comment (Koenkk) [reply test/docgen.test.js:10]: Screenshot 2021-10-14 at 20 02 14

[the remark this answers is not in the document]
Comment (Koenkk) [reply test/docgen.test.js:10]: I see, but still 46 seconds is long, lets just remove this test

**Pull request merged**